FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/CheckKeysFragment.kt

```diff
@@ -380,6 +396,8 @@ class CheckKeysFragment : BaseFragment<FragmentCheckKeysBinding>() {
   }
 
   companion object {
+    private const val REQUEST_CODE_SINGLE_PARTIALY_ENCRTYPRED_KEY = 1
```

Comment:
`REQUEST_CODE_SINGLE_PARTIALY_ENCRTYPRED_KEY` -> `REQUEST_CODE_SINGLE_PARTIALLY_ENCRYPTED_KEY`

Reply:
Interesting... Android Studio didn't show a warning about that. But usually shows. Thank you, I will fix it!